docs/How_to_Protect_AI_Models_in_Cloud_Native_Environments.md

```diff
@@ -36,20 +37,38 @@ To protect the model and its encryption key, the following preprocessing steps a
 
 The service supports attestation, measurement fetching and event log collecting of various platforms including Intel TDX, Trusted Platform Modules (TPM) and AMD Secure Encrypted Virtualization-Secure Nested Paging (SEV-SNP) (will be supported soon).
 
+CCNP is a good choice to get event log and it hides the complexity and is easy to use, sample code from CCNP:
+
+```Python
+from ccnp import Eventlog, EventlogType
+event_logs = Eventlog.get_platform_eventlog()
+```
+
+To verify that event log has not been tampered with, we can compare IMR (Integrated Measurement Register) value from CCNP fetching and replayed via event log.
```

Comment:
To verify that the event logs have not been tampered with, we can compare the measurement replayed from event logs with the IMR (Integrated Measurement Register) values fetched using CCNP.